### DISCUSSION TITLE
What is the itemsource on Avalonia?

### DISCUSSION THREAD

**Discussion by myplaths** [AvaloniaUI/Avalonia#7823] [Q&A] [answered]
For instance in wpf we have in a listview or in a combobox a property called itemsource in which we initialize a list through it using mvvm.

  <ComboBox  ItemsSource="{Binding Path=Persons}"   /> 

on Avalonia how do we do that?

**Comment by nielsbosma** [comment, marked as answer]
It's called "Items".

**Reply by robloo** [reply]
See #7553
